packages/ui-library/src/components/contacts-button/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
This is not going to be a reusable component right? It should go directly wherever its going to be used